### ISSUE TITLE
Wheel Chair Script - EMS

### ISSUE THREAD

**Issue by EMSBorat** [ioShinigami/SoloRP-Directory#157]
I saw that in Pillbox we have a non-functioning wheelchair that is just there for looks. I thought about the idea of having an actual script of renting out a wheelchair or crutches from Pillbox for patients that have broken their legs or can't walk. It would continue with the rp of helping them and then giving them it to get around until the injury is fully healed. My only concern is with everything added there are those who will abuse this and there will be many abandoned wheelchairs and crutches around the city. 
![wheelchair](https://user-images.githubusercontent.com/84989701/120026948-a4c8d180-bfa7-11eb-891b-0a86b05a06e5.jpg)

Here's an image of what I found online.

**Comment by Switty6** [comment]
Yep. This is going to be integrated as a part of the new medical system.
